pycqed/instrument_drivers/meta_instrument/qubit_objects/CCL_Transmon.py

```diff
@@ -3680,6 +3707,116 @@ def measure_gst_1Q(self,
         a = ma2.GST_SingleQubit_DataExtraction(label='Single_qubit_GST')
         return a
 
+    def make_qubit_dac_arc(self, dac_values=None, polycoeffs=None, MC=None,
+                           nested_MC=None, fluxChan=None):
+        """
+        Creates a qubit DAC arc by fitting a polynomial function through qubit
+        frequencies obtained by spectroscopy.
+
+        Arguments:
+        - dac_values: DAC values that are to be probed
+        - polycoeffs: initial coefficients of a second order polynomial.
```

Comment:
Maybe add that these are the polycoeffs that are used to predict the frequencies of the flux arc. 
Another (related comment) is that we should be able to get the polycoeffs from the qubit object based on the best knowledge of the system. I can explain how this should already be possible using existing parameters.